sql/catalyst/src/main/scala/org/apache/spark/sql/OutputMode.java

```diff
@@ -15,9 +15,10 @@
  * limitations under the License.
  */
 
-package org.apache.spark.sql.catalyst.analysis
+package org.apache.spark.sql;
 
-sealed trait OutputMode
-
-case object Append extends OutputMode
-case object Update extends OutputMode
+public enum OutputMode {
+  Append,
+  Update,
```

Comment:
This actually raises a good question.  I'm not sure if we can use enums here as I think that we need to have a notion of a `key` in order to do an `Update` mode.